### PR TITLE
User unfollow button changed to neutral colour

### LINKED ISSUE #10774: Change User Unfollow Button to Neutral Color
The user `Follow` and `Unfollow` buttons have the same color and this can be confusing for contributors. 

Screenshot 2022-03-10 at 09 59 57

A grey color like this would show the difference between the two buttons.

Screenshot 2022-03-10 at 10 01 40

Relevant code for this is in:
https://github.com/publiclab/plots2/blob/cc104ae472ddbb18aa94cc7f0ade0fa455c23cff/app/views/users/profile.html.erb#L127-L131

### PR DESCRIPTION
**Pull request by garimavs**
Worked on the issue mentioned below to change the user unfollow button from blue to neutral
Fixes #10774  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in a uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -127,7 +127,7 @@
           <% if current_user.following?(@profile_user) %>
             <a id="profile-unfollow-button" class="btn btn-primary" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
           <% else %>
-            <a id="profile-follow-button" class="btn btn-primary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%= @profile_user.id %>"><%= translation('users.profile.follow') %></a>
+            <a id="profile-follow-button" class="btn btn-secondary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%= @profile_user.id %>"><%= translation('users.profile.follow') %></a>
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:130]: @garimavs It's the unfollow button that should have a different color. You can use the class `btn-light`.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/users/profile.html.erb (hunk 1 above)
Comment (garimavs) [reply app/views/users/profile.html.erb:130]: Thanks for highlighting. I used the class "btn-secondary" to have a grey shade even when not hovering, but will update that too!

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -125,7 +125,7 @@
         <hr />
         <div id="follow_form" class="col-md-offset-4">
           <% if current_user.following?(@profile_user) %>
-            <a id="profile-unfollow-button" class="btn btn-primary" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
+            <a id="profile-unfollow-button" class="btn btn-light" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
```
Comment (TildaDares) [inline app/views/users/profile.html.erb:128]: I'm really sorry for the incovenience @garimavs, you are right about the button color. Can you switch it back to `btn-secondary`. Thanks!

**Comment by TildaDares** [comment]
@garimavs You also have to change the color here:
https://github.com/publiclab/plots2/blob/cc104ae472ddbb18aa94cc7f0ade0fa455c23cff/app/views/relationships/create.js.erb#L3

File: app/views/users/profile.html.erb (hunk 2 above)
Comment (garimavs) [reply app/views/users/profile.html.erb:128]: No worries! I'll do it again

**Comment by garimavs** [comment]
> @garimavs You also have to change the color here:
> 
> https://github.com/publiclab/plots2/blob/cc104ae472ddbb18aa94cc7f0ade0fa455c23cff/app/views/relationships/create.js.erb#L3

Thanks for mentioning it! Will do it right away!

**Comment by TildaDares** [comment]
Congratulations @garimavs 🎉 and welcome to Public Lab.

**Comment by garimavs** [comment]
> Congratulations @garimavs 🎉 and welcome to Public Lab.

Thanks, @TildaDares, learned a lot! Would love to contribute even in the future.

**Pull request merged**